### PR TITLE
Reorder axes

### LINKED ISSUE #29: [FEATURE] post_image reorder axis
**Is your feature request related to a problem? Please describe.**
The default **get_Image** return *tzxyc*, and the default **post_image** expects *xyzct*

**Describe the solution you'd like**
In  **get_Image** you can specify if *xyzct=False/True*, I was thinking that it would be convenient if the **post_image** would have something similar and thus would take care of the axis re-ordering
For example, giving the input dimension order as a string argument, and the **post_image** takes care of the re-ordering.

**Describe alternatives you've considered**
Take care of reordering between get_image and post_image, using np.moveaxis

### PR DESCRIPTION
**Pull request by erickmartins**
## Description

This adds the `dim_order` parameter to `get_image` and `post_image`. For `get_image`, it is a string specifying an ordering for axes to be returned by the function, and for `post_image` it specifies the axes order the numpy array has been passed on. Thjs should close #29.

## Checklist

Docstrings, unit tests and flake8 pass are done.

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.

### CODE REVIEW

**Comment by erickmartins** [comment]
(note that this set of changes also includes the ones in PR #45 - when that is merged, I will merge main into this branch)

File: ezomero/_gets.py [hunk 1]
```diff
@@ -61,9 +65,10 @@ def get_image(conn, image_id, no_pixels=False, start_coords=None,
 
     Notes
     -----
-    Regardless of whether `xyzct` is `True`, the numpy array is created as
-    TZYXC, for performance reasons. If `xyzct` is `True`, the returned `pixels`
-    array is actually a view of the original TZYXC array.
+    Regardless of whether `xyzct` is `True` or `dim_order` is set, the numpy
+    array is created asTZYXC, for performance reasons. If `xyzct` is `True`
```
Comment (mellertd) [inline ezomero/_gets.py:69]: minor typo here "asTZYXC" is missing a space

File: ezomero/_gets.py [hunk 2]
```diff
@@ -104,6 +109,13 @@ def get_image(conn, image_id, no_pixels=False, start_coords=None,
         if type(pyramid_level) is not int:
             raise TypeError('pyramid_level must be an int')
 
+    if dim_order is not None:
+        if type(dim_order) is not str:
+            raise TypeError('dim_order must be a str')
+        if set(dim_order.lower()) != set('xyzct'):
+            raise ValueError('dim_order must contain letters \
```
Comment (mellertd) [inline ezomero/_gets.py:116]: this error message has a newline in it, so it print's really weird when throwing an exception. You don't actually need the `\` there...you should just have two closed strings in the parenthesis like:

```
raise ValueError('dim_order must contain letters'
                 'xyzct exactly once')
```

Comment (mellertd) [reply ezomero/_gets.py:116]: except I am missing a space!

**Comment by mellertd** [comment]
Merging, but we'll have to resolve the coverage issue

**Pull request merged**